uaclient/files.py

```diff
@@ -116,6 +116,16 @@ def read(self) -> Optional[DOFType]:
 
         return self.data_object_cls.from_dict(parsed_data)
 
+    def try_read(self) -> Optional[DOFType]:
```

Comment:
Do we need this method ? It seems that the `read` method already returns None if the file is not found. We could add the other additional exceptions on that method as well in my opinion

Reply:
Good point. In the replacement PR, I instead refactored `read` to just return None if any exceptions occur